app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -31,7 +31,6 @@ def plan(capsule_content, options = {})
               end
               plan_action(ConfigureCapsule, capsule_content, environment, content_view, repository)
               sync_repos_to_capsule(capsule_content, repository_ids)
-              plan_action(RemoveOrphans, :capsule_id => capsule_content.capsule.id)
```

Comment:
@jlsherrill it seems to me that this happens already in the rake task. The `RemoveOrphans` action does not even use the `:capsule_id` param, [it seems](https://github.com/Katello/katello/blob/master/app/lib/actions/katello/capsule_content/remove_orphans.rb#L10).